add LAMMS pair_style to README (#57)
mir-group/nequip: README.md

A more in-depth introduction to the internals of NequIP can be found in the [tutorial notebook](https://deepnote.com/project/2412ca93-7ad1-4458-972c-5d5add5a667e).

> ## LAMMPS Integration 

Reviewer: ```suggestion
### LAMMPS Integration 
```

NequIP is integrated with the popular Molecular Dynamics code [LAMMPS](https://www.lammps.org/) which allows for MD simulations over large time- and length-scales and gives users access to the full suite of LAMMPS features. 

> The interface is implemented as the `pair_style nequip`. Usage requires two simple steps: 

Reviewer: ```suggestion
The interface is implemented as `pair_style nequip`. Using it requires two simple steps: 
```

> 1. Deploy a trained NequIP model. This step converts a NequIP model trained with PyTorch to [TorchScript](https://pytorch.org/docs/stable/jit.html). 
> This converts your model to an optimized, exported file that no longer has any Python dependency and ca be run from a standalone C++ program such as e.g. LAMMPS: 

Reviewer: ```suggestion
1. Deploy a trained NequIP model. This step compiles a NequIP model trained in Python to [TorchScript](https://pytorch.org/docs/stable/jit.html). 
The result is an optimized model file that has no Python dependency and can be used by standalone C++ programs such as LAMMPS: 
```

```

> nequip-deploy build path/to/example/run path/to/deployed.pth

Reviewer: ```suggestion
nequip-deploy build path/to/training/session/ path/to/deployed.pth
```

```

> 2. Change the LAMMPS input file to the nequip `pair_stlyle` and point it to the deployed NequIP model. 

Reviewer: ```suggestion
2. Change the LAMMPS input file to the nequip `pair_stlyle` and point it to the deployed NequIP model:
```

```
pair_style	nequip
pair_coeff	* * deployed.pth
```

> For Installation instruction, please see the NequIP pair_style repo here: 
> 
> ```
> github.com/mir-group/pair_nequip
> ```

Reviewer: ```suggestion
For installation instructions, please see the NequIP `pair_style` repo at https://github.com/mir-group/pair_nequip.
```
